src/__smoke-test__/smoke-test.spec.ts

```diff
@@ -0,0 +1,446 @@
+import { test, expect } from '@playwright/test';
+import {
+    HTTP_EVENT_TYPE,
+    JS_ERROR_EVENT_TYPE,
+    LCP_EVENT_TYPE,
+    FID_EVENT_TYPE,
+    PERFORMANCE_NAVIGATION_EVENT_TYPE,
+    PERFORMANCE_RESOURCE_EVENT_TYPE,
+    PAGE_VIEW_EVENT_TYPE,
+    SESSION_START_EVENT_TYPE
+} from '../plugins/utils/constant';
+
+// Secrete constant values retrieved from GitHub
+const endpoint = '${process.env.ENDPOINT}';
+const monitorId = '${process.env.MONITOR_ID}';
```

Comment:
question: How was the app monitor created? Are we assuming that it already exists?

Reply:
As mentioned before, for the scope of smoke tests, we do not need to create new application monitors for every run, as we can filter the events retrieved from CP by a timestamp.